toolkits/springer/packages/springer-listing/view/springer-listing.hbs

```diff
@@ -29,6 +29,11 @@
 				{{/each}}
 			</ul>
 			{{/if}}
+			{{#if content}}
```

Comment:
a tentative little 'um' here - wondering if a generic variable like `content` would conflict with other things? 

I'm a big fan of name-spacing things - so this might just be a 'me thing'.

Reply:
Hmm. The `content` name would be localised to the item of data, so I think we're good.